src/components/login-form.jsx

```diff
@@ -0,0 +1,85 @@
+import { Form, Formik } from "formik";
+import { Button, Col, Container, Row } from "react-bootstrap";
+import * as yup from "yup";
+import { useDispatch } from "react-redux";
+
+import "bootstrap/dist/css/bootstrap.min.css";
+
+import { save } from "../features/user/user-slice";
+import userService from "../services/api/user-service";
+import LoginFormTextField from "./login-form-text-field";
+
+const schema = yup.object({
+    email: yup.string().email("Invalid email address").required("Required"),
+    password: yup.string().required("Required"),
+});
+
+export default function LoginForm() {
+    const dispatch = useDispatch();
+
+    const handleLogin = (values) =>
+        userService.login(values).then((response) => dispatch(save(response)));
+
+    return (
+        <Row>
+            <Col>
+                <Formik
+                    validationSchema={schema}
+                    onSubmit={(values) => handleLogin(values)}
+                    initialValues={{
+                        email: "lucas.douglas@example.org",
```

Comment:
Nije dobra praksa da se hardcode-uju vrednosti na ovaj nacin. Kada imas ovako neke konstante, to je najbolje izdvojiti bas u neke `const` varijable. Idealno bi bilo da postoji poseban JS fajl koji ce sadrzati sve konstante koje imas u aplikaciji.
Vidim da imas `constants/api.js` fajl za jednu konstantu, pa bi mogao ili taj fajl da iskoristis da smestis sve konstante (ali da ga preimenujes u `index.js` npr.) ili da napravis novi fajl `index.js` u tom constants folderu.

Reply:
Ostavio sam `constants/api.js` ali sam dodao i `constants/index.js` sa `DEFAULT_EMAIL` i `DEFAULT_PASSWORD` konstantama.